app/lib/cfe_connection.rb

```diff
@@ -13,8 +13,7 @@ def status
   end
 
   def state_benefit_types
-    response = cfe_connection.get("state_benefit_type")
-    response.body.map { _1["name"] }.uniq
```

Comment:
I think we still need a `uniq` - otherwise Child Benefit crops up twice in our list. Maybe it needs to move into BenefitModel#benefit_list?

Reply:
I have moved the `uniq` to the `benefit_model` and specifically to the `benefit_list` method used to determine what to display. I think this method should just get whatever CFE sends it and pass that on... I also just properly read the second sentence, so yes i agree. 🤦